RiotSwiftUI/Modules/UserSessions/UserOtherSessions/View/UserOtherSessionsToolbar.swift

```diff
@@ -80,35 +80,33 @@ struct UserOtherSessionsToolbar: ToolbarContent {
     }
     
     private func optionsMenu() -> some View {
```

Comment:
While we're here…

```suggestion
    private var optionsMenu: some View {
```

Reply:
I don't want to trigger new actions just for that 🙈